src/queries/__tests__/role.test.tsx

```diff
@@ -696,6 +696,24 @@ describe('error messages', () => {
       `"Unable to find an element with role: "button", disabled state: true"`
     );
   });
+
```

Comment:
Can we have a test with both accessibilityState and accessibilityValue?

Reply:
I've added some more tests in this area.